### PR TITLE
Update intro.json

### PR DESCRIPTION
**Pull request by MuhammadZariyanAsif123**
The phrase has been updated 

from

This project will cover concepts like the map() method, find() method, parseInt(), the includes() method."

to

This project will cover concepts like the map(), find(), parseInt(), and includes() methods."

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [ ] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Comment by MuhammadZariyanAsif123** [comment]
The bug has been resolved.

Updated Phrase:

This project will cover concepts like the map(), find(), parseInt(), and includes() methods."

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -362,7 +362,7 @@
         "intro": [
           "Functional Programming is a popular approach to software development. In Functional Programming, developers organize code into smaller functions, then combine those functions to build complex programs.",
           "In this spreadsheet application project, you'll learn about parsing and evaluating mathematical expressions, implementing spreadsheet functions, handling cell references, and creating interactive web interfaces. You'll learn how to dynamically update the page based on user input.",
-          "This project will cover concepts like the <code>map()</code> method, <code>find()</code> method, <code>parseInt()</code>, the <code>includes()</code> method."
+          "This project will cover concepts like the <code>map()</code>, <code>find()</code>, <code>parseInt()</code> and <code>includes()</code> methods."
```
Comment (naomi-lgbt) [inline client/i18n/locales/english/intro.json:365]: `parseInt()` is not a method.

Comment (MuhammadZariyanAsif123) [reply client/i18n/locales/english/intro.json:365]: It’s a method which converts the string into an integer.

Comment (ilenia-magoni) [reply client/i18n/locales/english/intro.json:365]: no, it's a function
methods are those that you use on other values

Comment (MuhammadZariyanAsif123) [reply client/i18n/locales/english/intro.json:365]: @ilenia-magoni yeah you are right.
Thanks for the correction.

**Pull request closed without merging**